lib/ruby_llm/models.rb

```diff
@@ -66,7 +66,7 @@ def resolve(model_id, provider: nil, assume_exists: false, config: nil) # ruboco
             id: model_id,
             name: model_id.tr('-', ' ').capitalize,
             provider: provider_instance.slug,
-            capabilities: %w[function_calling streaming],
+            capabilities: provider_class.capabilities.capabilities_for(model_id),
```

Comment:
This will not work. The entire point of `assume_model_exists` is that _the model doesn't exist_. Take a look at the array of possible capabilities in `models_schema.json` instead.